WordPress/Classes/ViewRelated/Post/PostListViewController.swift

```diff
@@ -234,6 +234,28 @@ class PostListViewController : AbstractPostListViewController, UIViewControllerR
         }
     }
 
+    // MARK: - Data Model Interaction
+
+    /// Retrieves the post object at the specified index path.
+    ///
+    /// - Parameter indexPath: the index path of the post object to retrieve.
+    ///
+    /// - Returns: the requested post.
+    ///
+    private func postAtIndexPath(indexPath: NSIndexPath) -> Post {
+        guard let post = tableViewHandler.resultsController.objectAtIndexPath(indexPath) as? Post else {
+            // Retrieving anything other than a post object means we have an App with an invalid
```

Comment:
Ah, never mind, as it may receive a valid index but not return an expected `Post` object.